Probesys/bileto
# imp: Redesign the lists of organizations

## Related issue(s)



https://github.com/Probesys/bileto/issues/18

## Changes



- provide new `.list--padded` and `.list--strip` CSS classes, rename `.list--no-style` in `.list--nostyle`
- redesign the list of orgas in the admin so it will be easier to add actions
- add the count of the orgas in the admin
- redesign the list of orgas on the home page so it uses only reusable CSS components

## How to test manually



- go to the home page → check you see sub-organizations correctly and you can click on them
- go to the admin, organizations section → check you can see all the organizations and you edit them

## Checklist
